src/freenet/l10n/freenet.l10n.en.properties

```diff
@@ -122,36 +122,36 @@ Bookmarks.Defaults.Description.Flog.Bombe=Bombe's flog
 Bookmarks.Defaults.Description.Flog.Xor=Flog of the main Freetalk & Web Of Trust developer
 Bookmarks.Defaults.Description.Flog.Operhiem1=Flog of the Freenet release manager
 Bookmarks.Defaults.Description.Flog.ArneBab=ArneBab's notes
-Bookmarks.Defaults.ShortDescription.Linkageddon=Links to every Freenet website, sorted by when they were last updated, including some very offensive sites. Be careful what you click on!
-Bookmarks.Defaults.ShortDescription.Nerdageddon=Similar to Linkageddon but with the most offensive content removed
+Bookmarks.Defaults.ShortDescription.Linkageddon=All freesites, sorted by last updated. Includes boring and also very offensive sites. Be careful what you click on!
```

Comment:
`All freesites` is false, the `every` in the original does not imply `all` but `no filtering applied`. `Large collection of freesites` is more suitable.